cypress/support/commands.js

```diff
@@ -32,10 +32,53 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
```

Comment:
The `cy.get` command should be returned to ensure that the command chain is correctly continued. Consider adding a `return` statement before `cy.get` to maintain the chainability of Cypress commands.